client/src/store/new-request-module.ts

```diff
@@ -937,9 +933,7 @@ export class NewRequestModule extends VuexModule {
   get nrNames () {
```

Comment:
revert these changes please. I think this is what is causing the check mark to display on the wrong line.